### PR TITLE
qmmp 1.1.8 (new formula)

### PR DESCRIPTION
**Pull request by ibrokemypie**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/qmmp.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  depends_on "libbs2b"
+  depends_on "libsndfile"
+  depends_on "musepack"
+  depends_on "taglib"
+  depends_on "libmms"
+  depends_on "libsamplerate"
+
+  head do
+    url "http://svn.code.sf.net/p/qmmp-dev/code/branches/qmmp-1.2/"
+  end
```
Comment (bfontaine) [inline Formula/qmmp.rb]: Use `head "http://svn.code.sf.net/p/qmmp-dev/code/branches/qmmp-1.2/"` at the top of the file instead.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/qmmp.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  depends_on "libmms"
+  depends_on "libsamplerate"
+
+  def install
+    system "cmake", "./", "-USE_SKINNED", "-USE_ENCA", "-USE_QMMP_DIALOG", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    system bin/"qmmp", "--version"
```
Comment (dunn) [inline Formula/qmmp.rb]: It'd be nice to have a test that checks more of the program's functionality.

Comment (ibrokemypie) [inline Formula/qmmp.rb]: at the moment it seems that all other commands require spawning a qt window, which will always fail on the buildbot

Comment (dunn) [inline Formula/qmmp.rb]: Gotcha, thanks!

**Comment by dunn** [comment]
Merged as 5f7c78e540c77475615913e193351e78227c49d2.

**Pull request closed without merging**